apps/nova/src/components/dashboard/StudentOverview.tsx

```diff
@@ -2,30 +2,34 @@ import { useFormatMessage } from "@litespace/luna/hooks/intl";
 import { StudentOverview as Overview } from "@litespace/luna/StudentOverview";
 import { Typography } from "@litespace/luna/Typography";
 import { useFindPersonalizedStudentStats } from "@litespace/headless/student";
+import { useMediaQuery } from "@litespace/headless/mediaQuery";
 
 export const StudentOverview: React.FC = () => {
   const intl = useFormatMessage();
   const statsQuery = useFindPersonalizedStudentStats();
+  const mq = useMediaQuery();
 
   return (
-    <div className="grid gap-6 justify-items-start w-full">
+    <div className="grid gap-4 sm:gap-6 sm:my-0 justify-items-start w-full">
       <Typography
-        element="subtitle-2"
+        element={mq.sm ? "subtitle-2" : "body"}
         weight="bold"
         className="text-natural-950"
       >
         {intl("student-dashboard.overview.title")}
       </Typography>
 
-      <Overview
-        tutorCount={statsQuery.data?.tutorCount || 0}
-        completedLessonCount={statsQuery.data?.completedLessonCount || 0}
-        totalLearningTime={statsQuery.data?.totalLearningTime || 0}
-        totalLessonCount={statsQuery.data?.completedLessonCount || 0}
-        loading={statsQuery.isPending}
-        error={statsQuery.isError}
-        retry={statsQuery.refetch}
-      />
+      <div className="tw-flex tw-justify-center tw-items-center tw-w-full">
```

Comment:
I believe we don't need this div. I removed it and was not able to notice any difference.